.github/workflows/release.yml

```
@@ -0,0 +1,110 @@
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Set up JDK 8
+      uses: actions/setup-java@v4
+      with:
+        java-version: '8'
+        distribution: 'temurin'
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)